From gregkh@suse.de Thu May 15 21:04:53 2008
X-Envelope-From: <stable-commits-owner@vger.kernel.org>
X-Envelope-To: <vger@sl-h.de>
X-Delivery-Time: 1210881088
X-UID: 237685
Return-Path: <stable-commits-owner@vger.kernel.org>
X-RZG-CLASS-ID: mi
Received: from vger.kernel.org ([209.132.176.167])
	by mailin.webmailer.de (christine mi55) (RZmta 16.35)
	with ESMTP id T055a0k4FJpKSt for <vger@sl-h.de>;
	Thu, 15 May 2008 21:51:27 +0200 (MEST)
	(envelope-from: <stable-commits-owner@vger.kernel.org>)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1761420AbYEOTvB (ORCPT <rfc822;vger@sl-h.de>);
	Thu, 15 May 2008 15:51:01 -0400
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S1761378AbYEOTvB
	(ORCPT <rfc822;stable-commits-outgoing>);
	Thu, 15 May 2008 15:51:01 -0400
Received: from mx1.suse.de ([195.135.220.2]:48720 "EHLO mx1.suse.de"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1761356AbYEOTvA (ORCPT <rfc822;stable-commits@vger.kernel.org>);
	Thu, 15 May 2008 15:51:00 -0400
Received: from Relay2.suse.de (relay-ext.suse.de [195.135.221.8])
	(using TLSv1 with cipher DHE-RSA-AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mx1.suse.de (Postfix) with ESMTP id DA599419C9;
	Thu, 15 May 2008 21:50:59 +0200 (CEST)
Subject: patch usb-option-add-new-dell-5520-hsdpa-variant.patch added to 2.6.25-stable tree
To: dcbw@redhat.com,
 gregkh@suse.de
Cc: <stable@kernel.org>,
 <stable-commits@vger.kernel.org>
From:	<gregkh@suse.de>
Date:	Thu, 15 May 2008 12:04:53 -0700
In-Reply-To: <200805141920.m4EJKuWq019765@hera.kernel.org>
Message-Id: <20080515195059.14B6D14540CB@imap.suse.de>
Sender:	stable-commits-owner@vger.kernel.org
Precedence: bulk
Reply-To: linux-kernel@vger.kernel.org
List-ID: <stable-commits.vger.kernel.org>
X-Mailing-List:	stable-commits@vger.kernel.org
Status: R
X-Status: NC
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  


This is a note to let you know that we have just queued up the patch titled

     Subject: USB: option: add new Dell 5520 HSDPA variant

to the 2.6.25-stable tree.  Its filename is

     usb-option-add-new-dell-5520-hsdpa-variant.patch

A git repo of this tree can be found at 
    http://www.kernel.org/git/?p=linux/kernel/git/stable/stable-queue.git;a=summary


>From stable-bounces@linux.kernel.org Wed May 14 12:21:22 2008
From: Dan Williams <dcbw@redhat.com>
Date: Wed, 14 May 2008 19:20:56 GMT
Subject: USB: option: add new Dell 5520 HSDPA variant
To: jejb@kernel.org, stable@kernel.org
Message-ID: <200805141920.m4EJKuWq019765@hera.kernel.org>

From: Dan Williams <dcbw@redhat.com>

commit 96cb15cf977356d9d3117dd88f3fe187d6024f4b upstream

New variant of the 5520 found by Luke Sheldrick.

Signed-off-by: Dan Williams <dcbw@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/option.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -293,6 +293,7 @@ static struct usb_device_id option_ids[]
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8133) }, /* Dell Wireless 5720 == Novatel EV620 CDMA/EV-DO */
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8136) },	/* Dell Wireless HSDPA 5520 == Novatel Expedite EU860D */
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8137) },	/* Dell Wireless HSDPA 5520 */
+	{ USB_DEVICE(DELL_VENDOR_ID, 0x8138) },	/* Dell Wireless 5520 Voda I Mobile Broadband (3G HSDPA) Minicard */
 	{ USB_DEVICE(ANYDATA_VENDOR_ID, ANYDATA_PRODUCT_ADU_E100A) },
 	{ USB_DEVICE(ANYDATA_VENDOR_ID, ANYDATA_PRODUCT_ADU_500A) },
 	{ USB_DEVICE(AXESSTEL_VENDOR_ID, AXESSTEL_PRODUCT_MV110H) },


Patches currently in stable-queue which might be from dcbw@redhat.com are

queue-2.6.25/usb-option-add-new-dell-5520-hsdpa-variant.patch
--
To unsubscribe from this list: send the line "unsubscribe stable-commits" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

